Source/ablastr/fields/MultiFabRegister.cpp

```diff
@@ -365,8 +364,7 @@ namespace ablastr::fields
     ) const
     {
         if (m_mf_register.count(internal_name) == 0) {
-            // FIXME: temporary, throw a std::runtime_error
-            // throw std::runtime_error("MultiFabRegister::get name does not exist in register: " + internal_name);
+            throw std::runtime_error("MultiFabRegister::get name does not exist in register: " + internal_name);
             return nullptr;
```

Comment:
I think we can remove this:
```suggestion
            throw std::runtime_error("MultiFabRegister::get name does not exist in register: " + internal_name);
```